### DISCUSSION TITLE
Markdown to Neorg exporter?

### DISCUSSION THREAD

**Discussion by Gaulm** [nvim-neorg/neorg#428] [Q&A]
Is it possible to convert a Markdown file to Neorg automatically? Manual conversion is not really a thing I'd do with thousands of lines, even a partial or semi functional conversion would be great.

**Comment by tristone13th** [comment]
I second, also need norg to md.

**Reply by max397574** [reply]
that already exists

**Comment by mrossinek** [comment]
You can export a norg file to markdown by doing the following:
1. Load the following modules as part of your neorg configuration:
```lua
        ["core.export"] = {},
        ["core.export.markdown"] = {
            config = {
                extensions = "all",
            },
        },
```
2. Open a norg file
3. Run the command `:Neorg export to-file my_file.md`

**Reply by Gaulm** [reply]
I need md to norg, not norg to md.

I have many documents written in markdown, but its treesitter implementation is slow on large files and switching to neorg would be an improvement. But I can't convert manually because of how big the files are, it would take me several days.

**Reply by mrossinek** [reply]
Oh sorry I misread based on the other reply..
This does not exist at the moment. But it is definitely planned!

**Comment by pysan3** [comment]
https://github.com/pysan3/minorg

Created a tool!
